sql/moz-fx-data-shared-prod/mozilla_org_derived/ga_clients_v1/metadata.yaml

```diff
@@ -4,6 +4,10 @@ description: |-
   Key is ga_client_id.
 owners:
 - frank@mozilla.com
+scheduling:
+  depends_on_past: true
```

Comment:
If it were me, to avoid confusion I'd put the DAG names back in commented out, and with an explanatory comment to clarify these are intentionally no longer scheduled.

Reply:
Good idea, will do